docs/documentation/integration-description.md

```diff
@@ -42,10 +42,17 @@ This is the contents of the `HelloWorld_description.md` file:
 - You can use the following API Key: `43ea9b2d-4998-43a6-ae91-aba62a26868c`
 ```
 
-To display the integration description, click on the question mark button in the integration configuration panel
+### Cortex XSOAR versions up to 6.0
```

Comment:
```suggestion
### Cortex XSOAR versions up to 6.0 (not including)
```

Reply:
I kept 6.0 and added "including" in the text